docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,19 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.0.1
+2020-11-10
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
+
+### Upgrades
+
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.2](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+
+### Bug fixes and minor changes
+- Fix segfault in Virtual Box version check on Big Sur. See [docker/for-mac#4997](https://github.com/docker/for-mac/issues/4997).
+
 ## Docker Desktop Community 2.5.0.0
 2020-11-02
 
```

Comment:
Can we update the download URL for 2.5.0.0?

Reply:
This is https://desktop.docker.com/mac/stable/49427/Docker.dmg

Reply:
```
> [Download](https://desktop.docker.com/mac/stable/49427/Docker.dmg)
```

Reply:
Done!